samples/sample_encode/src/pipeline_encode.cpp

```diff
@@ -876,10 +877,18 @@ mfxStatus CEncodingPipeline::AllocFrames()
     MSDK_ZERO_MEMORY(VppRequest[0]);
     MSDK_ZERO_MEMORY(VppRequest[1]);
 
+    mfxU16 tmpTargetKbps = m_mfxEncParams.mfx.TargetKbps;
+
     // Querying encoder
     sts = GetFirstEncoder()->Query(&m_mfxEncParams, &m_mfxEncParams);
     MSDK_CHECK_STATUS(sts, "Query (for encoder) failed");
 
+    if (MFX_WRN_INCOMPATIBLE_VIDEO_PARAM == sts && tmpTargetKbps != m_mfxEncParams.mfx.TargetKbps)
+    {
+        msdk_printf(MSDK_STRING("WARNING: Target bitrate was changed from %d kbps to %d kbps due to default limitations.\n"), tmpTargetKbps, m_mfxEncParams.mfx.TargetKbps);
+        msdk_printf(MSDK_STRING("         To prevent this, use -BitrateLimit:off flag (see docs for additional details).\n\n"));
```

Comment:
Is BitrateLimit can be the only reason for TargetKbps adjustment? @GlaUgrub , can you please comment?

Reply:
1) AFAIK BitrateLimit is the only reason which can change bitrate value.
2) Value 'of "TargetKbps" can be changed by MSDK if pair ("TargetKbps", "BRCParamMultipier") was provided. Sometimes MSDK changes the multiplier (e.g. pair (10000, 20) can be changed to (40000, 5)). But bitrate value given by the pair remains unchanged.